### PR TITLE
[TECH-466] Switch configuration endpoints

### PR DESCRIPTION
**Pull request by av-mads**

### CODE REVIEW

File: lib/av_client.ts [hunk 1]
```diff
@@ -564,9 +553,9 @@ export class AVClient implements IAVClient {
       throw new InvalidStateError('Cannot challenge ballot, no user session')
     }
 
-   let attempts = 0;
-   
-   const executePoll = async (resolve, reject) => {
```
Comment (av-martin) [inline lib/av_client.ts:569]: This should have been indented in the other direction

File: lib/av_client.ts [hunk 2]
```diff
@@ -583,7 +572,7 @@ export class AVClient implements IAVClient {
       }
     };
   
-   return new Promise(executePoll);
```
Comment (av-martin) [inline lib/av_client.ts:586]: ^

File: lib/av_client.ts (hunk 1 above)
Comment (av-alexistoledo) [reply lib/av_client.ts:569]: You're right. I'll fix it right away

**Review by av-alexistoledo — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -31,23 +31,20 @@ To run tests:
 yarn run test
 ````
 
+To run benaloh flow and walkthrough test:
+
+Delete the `.skip` on line 9 (test/benaloh_flow.test.ts) and on lines 11 and 90 (test/walkthrough.test.ts) and make sure you have all the services in the devbox running.
```
Comment (av-lukas) [inline README.md:36]:
```suggestion
Delete the `.skip` instruction on line 9, in the test/benaloh_flow.test.ts, and on lines 11 and 90, in test/walkthrough.test.ts, and make sure you have Devbox’s services running and seeded.
```

File: README.md [hunk 4]
```diff
@@ -31,23 +31,20 @@ To run tests:
 yarn run test
 ````
 
+To run benaloh flow and walkthrough test:
```
Comment (av-lukas) [inline README.md:34]:
```suggestion
To run the Benaloh flow and walkthrough test:
```

File: lib/av_client/types.ts [hunk 5]
```diff
@@ -330,3 +264,151 @@ export type ReadableOptionSelection = {
   title: string
   text?: string
 }
+
+/** New stuff */
```
Comment (av-lukas) [inline lib/av_client/types.ts:268]:
```suggestion
```

File: lib/av_client/types.ts [hunk 6]
```diff
@@ -330,3 +264,151 @@ export type ReadableOptionSelection = {
...
+  affidavit?: AffidavitConfig
+  castRequestItemAttachmentEncryptionKey?: string
+}
+
+export interface LatestConfigItems {
+  thresholdConfig: ThresholdConfig;
+  voterAuthorizerConfig: VoterAuthorizer;
+  ballotConfigs: BallotConfigMap;
+  contestConfigs: ContestConfigMap;
+  // votingRoundConfigs: VotingRoundConfig[];
```
Comment (av-lukas) [inline lib/av_client/types.ts:283]: Do we need this around?

**Review by av-lukas — APPROVED** [review]

File: lib/av_client/types.ts (hunk 6 above)
Comment (av-alexistoledo) [reply lib/av_client/types.ts:282]: We're using it on the voting round branch that comes next to this one

**Pull request merged**